packages/boot/tools/drivers.ts

```diff
@@ -17,9 +17,9 @@ import type {
   CurrentWalletRecord,
   SmartWallet,
   UpdateRecord,
+  OfferSpec,
 } from '@agoric/smart-wallet/src/smartWallet.js';
 import type { WalletFactoryStartResult } from '@agoric/vats/src/core/startWalletFactory.js';
-import type { OfferSpec } from '@agoric/smart-wallet/src/offers.js';
```

Comment:
consider leaving the type where it was, even if the logic had to move

Reply:
done